tests/accelerators/test_accelerator_connector.py

```diff
@@ -546,27 +527,17 @@ def test_accelerator_gpu_with_devices(devices, plugin):
 
     trainer = Trainer(accelerator="gpu", devices=devices)
 
-    assert trainer.gpus == devices
+    assert trainer.devices == devices
     assert isinstance(trainer.strategy, plugin)
     assert isinstance(trainer.accelerator, GPUAccelerator)
 
 
 @RunIf(min_gpus=1)
 def test_accelerator_auto_with_devices_gpu():
     trainer = Trainer(accelerator="auto", devices=1)
-    assert isinstance(trainer.accelerator, GPUAccelerator)
```

Comment:
We should keep the accelerator instance type check, no?

Reply:
Yup

Reply:
I'll revert the test.